### PR TITLE
Add RSU Vendor Filter

### PR DESCRIPTION
**Pull request by mwodahl**
Adds an additional menu to the map to allow users to filter displayed RSUs by vendor. Additionally, updates the geospatial multi-rsu select functionality to only include RSUs with the chosen vendor.

Unit tests for the webapp and API have been updated to pass with these changes.

### CODE REVIEW

File: services/api/tests/src/test_rsu_geo_query.py [hunk 1]
```diff
@@ -93,3 +93,19 @@ def test_query_rsu_devices(mock_query_db):
 
     assert actual_result == ["10.11.81.12"]
     assert code == 200
+
+@patch("api.src.rsu_commands.pgquery.query_db")
+def test_query_rsu_devices_with_vendor(mock_query_db):
+    mock_query_db.return_value = [
+        ({"ip": "10.11.81.12"},),
+    ]
+    print(rsu_geo_query_data.point_list)
```
Comment (dmccoystephenson) [inline services/api/tests/src/test_rsu_geo_query.py]: Is this print statement necessary?

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
This looks good for the most part! A couple things to note:
- There is a seemingly unnecessary 'print' statement in one of the introduced unit tests. I have left a comment about this.
- Using Python 3.12.2, I am unable to install multidict 6.0.4. This has been addressed in https://github.com/CDOT-CV/jpo-cvmanager/pull/44 by updating the target version to 6.0.5, however, so I don't think this needs to be corrected in this PR.
- The services unit tests are passing. This was tested using a virtual environment, Python 3.12.2 & multidict 6.0.5.
- The unit tests for the webapp are passing.
- I have yet to test this functionality out in a browser.

File: services/api/tests/src/test_rsu_geo_query.py (hunk 1 above)
Comment (mwodahl) [reply services/api/tests/src/test_rsu_geo_query.py]: Nope - I've gone ahead and removed this. Thanks for catching that

**Review by dmccoystephenson — APPROVED** [review]
Looks good to me!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Review by drewjj — APPROVED** [review]
Looks good

**Pull request merged**